delphi-checks-testkit/src/main/java/au/com/integradev/delphi/builders/AbstractDelphiTestFile.java

```diff
@@ -55,6 +55,8 @@ protected InputFile inputFile() {
       file.deleteOnExit();
 
       try (FileWriter fileWriter = new FileWriter(file, UTF_8)) {
+        // Prepend utf-8 BOM
```

Comment:
I think you misunderstood me - my nit was that `utf-8` should be capitalised, especially if `BOM` is 😄

Reply:
Oops, I have a bias towards lowercase "utf-8" for some reason.